packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Writers/CodeWriter.cs

```diff
@@ -255,9 +255,10 @@ internal void WriteXmlDocs(XmlDocProvider? docs)
             }
         }
 
-        public void WriteProperty(PropertyProvider property)
+        public void WriteProperty(PropertyProvider property, bool isPublicContext = false)
```

Comment:
I see that in the code WriteProperty is mostly called with true: codeWriter.WriteProperty(property1, true);. When would it be called with false? (is there an example?)

Reply:
thats test code forcing a scenario the code flow call comes from TypeProviderWriter which passes in whether the property is public and all of its types is nested in are also public.